src/components/InForm/InForm.tsx

```diff
@@ -10,73 +10,64 @@ import {
   mergeStyleSets,
   Modal,
 } from "@fluentui/react";
-import { makeStyles } from "@fluentui/react-components";
-import React, { ChangeEvent, FormEvent, useState } from "react";
-import { PeoplePicker } from "../PeoplePicker/PeoplePicker";
+import React, { ChangeEvent, FormEvent, useContext, useState } from "react";
+import { PeoplePicker, SPPersona } from "../PeoplePicker/PeoplePicker";
 import { useBoolean } from "@fluentui/react-hooks";
 import { OFFICES } from "../../constants/Offices";
 import { GS_GRADES, NH_GRADES, MIL_GRADES } from "../../constants/GradeRanks";
 import { emptype, EMPTYPES } from "../../constants/EmpTypes";
 import { worklocation, WORKLOCATIONS } from "../../constants/WorkLocations";
 import {
+  makeStyles,
   Button,
   Input,
   InputOnChangeData,
   Label,
   Radio,
   RadioGroup,
   RadioGroupOnChangeData,
-  Switch,
-  SwitchOnChangeData,
+  Text,
 } from "@fluentui/react-components";
+import { UserContext } from "../../providers/UserProvider";
+import { INewInForm, RequestApiConfig } from "../../api/RequestApi";
 
-interface IInForm {
-  /** Required - Contains the Employee's Name */
-  empName: string;
-  /** Required - Employee's Type valid values are:
-   * 'civ' - for Civilian Employees
-   * 'mil' - for Military Employees
-   * 'ctr' - for Contracted Employees
-   */
-  empType: emptype;
-  /** Required - The Employee's Grade/Rank.  Not applicable if 'ctr' */
-  gradeRank: string;
-  /** Required - Possible values are 'local' and 'remote'  */
-  workLocation: worklocation;
-  /** Required - The Employee's Office */
-  office: string;
-  /** Required - Can only be 'true' if it is a New to USAF Civilain.  Must be 'false' if it is a 'mil' or 'ctr' */
-  isNewCiv: boolean;
-  /** Required - The user's previous organization.  Will be "" if isNewCiv is false */
-  prevOrg: string;
-  /** Required - The user's Estimated Arrival Date */
-  eta: Date;
-}
-
-/** For new forms, allow certain fields to be blank or undefined to support controlled components */
-interface INewInForm extends Omit<IInForm, "empType" | "workLocation" | "eta"> {
-  empType: emptype | "";
-  workLocation: worklocation | "";
-  eta: Date | undefined;
+export enum INFORMVIEWS {
+  COMPACT,
+  NEW,
+  EDIT,
 }
```

Comment:
Maybe add a quick jsdoc describing what this should be used for?

Reply:
yes, need to go through and clean up and add JSDoc throughout -- added it to this location for now